app/views/users/show.html.erb

```diff
@@ -24,15 +24,106 @@
     }
     <% end %>
   </style>
-  <div class="user-profile-header" style="<%= user_colors_style(@user) %>" itemscope itemtype="http://schema.org/Person" itemprop="mainEntity">
-    <meta itemprop="url" content="https://dev.to/<%= @user.username %>">
+
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Person",
+    "mainEntityOfPage": {
+      "@type": "WebPage",
+      "@id": "https://dev.to/<%= @user.username %>"
+    },
+    "url": "<%= user_url(@user) %>",
+    "image": <%= ProfileImage.new(@user).get(width: 320) %>,
+    "name": "<%= @user.name %>",
+    "description": [
+      {
+        "<%= @user.summary.presence || ["404 bio not found"].sample %>",
```

Comment:
this is missing the name of the key

Reply:
The name of the key is `description` here. 😊

Reply:
the `description` on line 39 is fine, bu this is technically a hash, see line 40, so it needs a key, but I think I've understood, this is supposed to be an array of strings right? so no, `{ }` is needed. I guess :D